src/main/java/com/dzaro/AddMealInterface.java

```diff
@@ -0,0 +1,15 @@
+package com.dzaro;
+
+public class AddMealInterface {
+    UserAnswer userAnswer = new UserAnswer();
+    MealDAO mealDAO = new MealDAO();
+    public void start() {
+        System.out.println();
+        try {
+            mealDAO.addMealAndIngriedientsForDB(userAnswer.collectMeal());
```

Comment:
Consider this function name:
```suggestion
            mealDAO.saveMeal(userAnswer.collectMeal());
```